Integrations/integration-EWSv2.yml

```diff
@@ -715,24 +716,39 @@ script:
     def fetch_emails_as_incidents(account, folder_name):
         start_time = EWSDateTime.now(tz=EWSTimeZone.timezone('UTC'))
         last_run = get_last_run()
-        last_emails = fetch_last_emails(account, folder_name, last_run.get(LAST_RUN_TIME), last_run.get(LAST_RUN_IDS))
-
-        ids = []
-        incidents = []
-        for item in last_emails:
-            if item.message_id:
-                ids.append(item.message_id)
-                incident = parse_incident_from_item(item)
-                incidents.append(incident)
-
-        new_last_run = {
-            LAST_RUN_TIME: start_time.ewsformat(),
-            LAST_RUN_FOLDER: folder_name,
-            LAST_RUN_IDS: ids
-        }
 
-        demisto.setLastRun(new_last_run)
-        return incidents
+        try:
+            last_emails = fetch_last_emails(account, folder_name, last_run.get(LAST_RUN_TIME), last_run.get(LAST_RUN_IDS))
+
+            ids = []
+            incidents = []
+            for item in last_emails:
+                if item.message_id:
```

Comment:
@BenJoParadise if there's no message_id this line won't raise an error? Asking because of the dot notation

Reply:
It's an object property. The object will always have this property, but it might be None, in which case the predicate will be false.